style.css

```diff
@@ -0,0 +1,552 @@
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-top: 5%;
+}
+
+.featured__spk {
+  font-size: 20px;
+  color: #272a31;
+  margin-top: 2%;
+}
+
+.spk__information__container {
+  display: flex;
+  background-image: url("./images/checked\ image.png");
+  background-repeat: no-repeat;
+  background-size: 54px 54px;
+}
+
+.spk__info__container {
+  display: flex;
+  flex-direction: column;
+  padding: 12px;
+}
+
+.checked__img {
+  width: 80px;
+  height: 80px;
+  margin-left: 5%;
+}
+
+.speaker__img {
+  width: 150px;
+  height: 150px;
+  margin-left: 20px;
+  margin-top: 20px;
+}
+
+.spk__name {
+  color: #272a31;
+  font-size: 20px;
+}
+
+.spk__title {
+  color: #ef513c;
+  font-size: 15px;
+  margin-top: 12px;
+  font-style: italic;
+}
+
+.spk__hr {
+  width: 20px;
+  margin-top: 12px;
+}
+
+.spk__role {
+  color: #272a31;
+  font-size: 15px;
+  margin-top: 12px;
+}
+
+.about {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  background-image: url("./images/bkgrnd.png");
+  background-repeat: no-repeat;
+  background-position: center center;
+  background-size: cover;
+  gap: 10px;
+  width: 100%;
+  height: 100vh;
+  padding: 12px;
+}
+
+.about__title {
+  text-align: center;
+  font-size: 20px;
+  color: #ef513c;
+}
+
+.about__subtitle {
+  text-align: center;
+  font-size: 35px;
+  color: #ef513c;
+}
+
+.about__description {
+  background-color: #d3d3d3;
+  border: 4px solid white;
+  color: #272a31;
+  padding: 1%;
+  width: 60%;
+  align-self: center;
+  margin-top: 20px;
+}
+
+.contact {
+  line-height: 20px;
+  font-size: 13px;
+  text-align: center;
+  color: #272a31;
+}
+
+.about__email {
+  text-decoration: underline;
+  color: #272a31;
+}
+
+.about__logo {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 12px;
+}
+
+.about__logo__title {
+  text-align: center;
+  font-size: 19px;
+  color: #272a31;
+}
+
+.about__logo__description {
+  text-align: center;
+  color: #272a31;
+  font-size: 15px;
+  margin-top: 20px;
+}
+
+.about__img {
+  margin-top: 30px;
+}
+
+.past__conference {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  position: relative;
+  padding: 12px;
+  margin-top: 30px;
+}
+
+.past__title {
+  text-align: center;
+  color: #272a31;
+}
+
+.past__description {
+  text-align: center;
+  color: #272a31;
+  margin-top: 20px;
+  font-size: 13px;
+}
+
+.past__cards {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+.past1 {
+  background-image: url("./images/cop27.jpg");
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: cover;
+  margin-top: 30px;
+  height: 32vh;
+  width: 40vw;
+}
+
+.past2 {
+  background-image: url("./images/cop27_18_11.jpg");
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: cover;
+  margin-top: 30px;
+  height: 32vh;
+  width: 40vw;
+}
```

Comment:
- [OPTIONAL] Kindly make sure to adjust the width of these cards in order to look as match as possible as the provided Behance design on the mobile version.

   - ### YOUR DESIGN
   
    ![image](https://user-images.githubusercontent.com/87197412/225733104-cb4e44c3-40f5-424e-b043-06a5b7883f03.png)

   - ### BEHANCE DESIGN
    ![image](https://user-images.githubusercontent.com/87197412/225733178-cb3016ee-dcaf-4a73-8bd5-867ad4921f68.png)

Reply:
I appreciate it will adjust the image size